packages/storybook/stories/CustomTeamsEndpoint.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import { DetailedBetaBanner } from './BetaBanners/DetailedBetaBanner';
+import CustomTeamsEndpointSnippetText from '!!raw-loader!./snippets/CustomTeamsEndpoint.snippet.tsx';
+
+<Meta
+  id="customTeamsEndpoint"
+  title="Concepts/Custom Teams Endpoint"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Custom Teams Endpoint
+
+<DetailedBetaBanner version={'0.9.0-beta.1'} />
+
+Custom Teams Endpoint is a flow where you can join an Azure Communications Service powered call as a Teams user.
+
+To learn more about the Custom Teams Endpoint, click [here](https://learn.microsoft.com/en-us/azure/communication-services/concepts/teams-endpoint)
+
+## Examples
+
+Leveraging custom teams endpoint, enables developers to create customized Teams experiences:
+
+- Browser support on mobile devices
+- User interface (UI) customization
+- End users can join calls as their M365 user instead of an External user.
+
+## How does UI Library interact with Custom Teams Endpoint
+
+The UI Library helps Contoso build custom teams client experiences via a simplified API surface, providing a set of UI components,
+and offering UI composites to reduce development time to production. The UI Library is able to offer all of these capabilities by taking
+in a teams access token as input and using teams-specific calling functions for making calls as a Teams user.
+
+Note: The UI Library is not able to create teams access tokens or interact with your existing tenant.
+
+## How to incorporate Custom Teams Endpoint with the UI Library in your experience.
+
+The following is a code snippet of how to create a custom teams endpoint experience with the Call Composite.
+
+<Source code={CustomTeamsEndpointSnippetText} />
+
+## FAQs
+
```

Comment:
Would be nice to point developers to the guide of generating a teams token in FAQ
https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/manage-teams-identity?pivots=programming-language-javascript